README.md

```diff
@@ -1,210 +1,119 @@
+These commands export information using the Ledger Exporter output files within a specified datastore (currently [datastore](https://github.com/stellar/go/tree/master/support/datastore) only supports GCS). This allows users to provide a start and end ledger range. The commands in this category export a list of everything that occurred within the provided range. All of the ranges are inclusive.
 
-<br>
+#### Common Flags
 
-### **export_ttl**
+| Flag           | Description                                                                                   | Default                 |
+| -------------- | --------------------------------------------------------------------------------------------- | ----------------------- |
+| start-ledger   | The ledger sequence number for the beginning of the export period. Defaults to genesis ledger | 2                       |
+| end-ledger     | The ledger sequence number for the end of the export range                                    | 0                       |
+| strict-export  | If set, transform errors will be fatal                                                        | false                   |
+| testnet        | If set, will connect to Testnet instead of Pubnet                                             | false                   |
+| futurenet      | If set, will connect to Futurenet instead of Pubnet                                           | false                   |
+| extra-fields   | Additional fields to append to output jsons. Used for appending metadata                      | ---                     |
+| captive-core   | If set, run captive core to retrieve data. Otherwise use TxMeta file datastore                | false                   |
+| datastore-path | Datastore bucket path to read txmeta files from                                               | ledger-exporter/ledgers |
+| buffer-size    | Buffer size sets the max limit for the number of txmeta files that can be held in memory      | 1000                    |
+| num-workers    | Number of workers to spawn that read txmeta files from the datastore                          | 5                       |
+| retry-limit    | Datastore GetLedger retry limit                                                               | 3                       |
+| retry-wait     | Time in seconds to wait for GetLedger retry                                                   | 5                       |
 
-```bash
-> stellar-etl export_ttl --end-ledger 500000 --output export_ttl.txt
-```
+> _*NOTE:*_ Using captive-core requires a Stellar Core instance that is v20.0.0 or later. The commands use the Core instance to retrieve information about changes from the ledger. These changes can be in the form of accounts, offers, trustlines, claimable balances, liquidity pools, or account signers.
```

Comment:
might be easier to link to ledger entry types in our stellar docs, versus enumerating each one individually.